lazyflow/operators/ioOperators/opNpyFileReader.py

```diff
@@ -25,11 +25,17 @@
 import numpy
 import copy
 
+import logging
+logger = logging.getLogger(__name__)
+
+
 class OpNpyFileReader(Operator):
     name = "OpNpyFileReader"
     category = "Input"
 
     FileName = InputSlot(stype='filestring')
+    InternalPath = InputSlot(stype='string', value="")
```

Comment:
With `InternalPath` as an optional InputSlot, the order in which 'FileName' and 'InternalPath' are set becomes important. This can be handled by `OpInputDataReader`. Is this behaviour something we can live with?